content/docs/overview/identity-and-access-management/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Identity and Access Management"
+description: "RKVST IAM Concepts"
+lead: "Setting Up Your RKVST Tenancy"
+date: 2021-06-14T10:57:58+01:00
+lastmod: 2021-06-14T10:57:58+01:00
+draft: false
+images: []
+menu: 
+  docs:
+    parent: "overview"
+weight: 5
+toc: true
+---
+
+## Tenancies and Accounts
+
+Each RKVST tenancy represents an organization, and each RKVST account represents an individual user. There may be multiple accounts within a tenancy if there are several members within an organization.
```

Comment:
Can we note that a user can be part of multiple tenancies? 

"Additionally, an individual user can be part of multiple tenancies."

Reply:
Added